galaxy_ng/app/api/v3/viewsets/collection.py

```diff
@@ -87,21 +92,71 @@ class CollectionViewSet(api_base.LocalSettingsMixin,
     serializer_class = CollectionSerializer
 
 
-class UnpaginatedCollectionVersionViewSet(api_base.LocalSettingsMixin,
-                                          ViewNamespaceSerializerContextMixin,
-                                          pulp_ansible_views.UnpaginatedCollectionVersionViewSet):
+class UnpaginatedCollectionVersionViewSet(
+    api_base.LocalSettingsMixin,
+    ViewNamespaceSerializerContextMixin,
+    DestroyModelMixin,
```

Comment:
this Mixin is in the wrong class, it would go to the Collection Version Viewset, but looks like is not needed.